subcommand/server-acl-init/command.go

```diff
@@ -46,6 +46,8 @@ type Command struct {
 	flagConsulCACert              string
```

Comment:
can remove variables:
* `flagReleaseName`
* `flagServerLabelSelector`
* `flagReplicas`

Reply:
👍 Ah that was probably me failing at git rebasing 😃